README.md

```diff
@@ -2,34 +2,34 @@
   <img src="http://dp.hanlon.io/0u2T0M3Z3i0g/shhh.png" height="150" width="300"/>
   <p>Jest Silent Reporter</p>
 </h1>
-<p align="center">Jest custom reporter that only prints failed tests.</p>
+<p align="center">Custom reporter for [Jest](https://jestjs.io/) that only prints failed tests.</p>
```

Comment:
@kivikakk this markdown link doesn't seem to be rendered correctly -- perhaps due to being wrapped in a `<p>` element. Do you think this could be a bug in [cmark-gfm](https://github.com/github/cmark-gfm)?

Reply:
Not a bug -- intended behaviour according to the CommonMark spec. See [§4.6 HTML blocks](https://spec.commonmark.org/0.29/#html-blocks) -- note the paragraph:

> Everything until the next blank line or end of document gets included in the HTML block. So, in the following example, what looks like a Markdown code block is actually part of the HTML block, which continues until a blank line or the end of the document is reached:

In other words, Markdown parsing is suspended while in an HTML block. You'll need to write the `<a>` out yourself here!

Reply:
➡️ #13

Reply:
Thanks for the clarification!